frontend/src/e2e-playwright/specs/4_finance/fee-decisions.spec.ts

```diff
@@ -0,0 +1,82 @@
+// SPDX-FileCopyrightText: 2017-2021 City of Espoo
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+import {
+  careArea2Fixture,
+  daycare2Fixture,
+  feeDecisionsFixture,
+  Fixture
+} from 'e2e-test-common/dev-api/fixtures'
+import {
+  insertEmployeeFixture,
+  insertFeeDecisionFixtures,
+  resetDatabase
+} from 'e2e-test-common/dev-api'
+import { newBrowserContext } from '../../browser'
+import config from 'e2e-test-common/config'
+import { Page } from 'playwright'
+import { initializeAreaAndPersonData } from 'e2e-test-common/dev-api/data-init'
+import EmployeeNav from 'e2e-playwright/pages/employee/employee-nav'
+import {
+  FinancePage,
+  FeeDecisionsPage
+} from 'e2e-playwright/pages/employee/finance/finance-page'
+import { waitUntilEqual } from 'e2e-playwright/utils'
+import { employeeLogin } from 'e2e-playwright/utils/user'
+
+let page: Page
+let feeDecisionsPage: FeeDecisionsPage
+
+beforeEach(async () => {
+  await resetDatabase()
+  const fixtures = await initializeAreaAndPersonData()
+  const careArea = await Fixture.careArea().with(careArea2Fixture).save()
+  await Fixture.daycare().with(daycare2Fixture).careArea(careArea).save()
+
+  await insertFeeDecisionFixtures([
+    feeDecisionsFixture(
+      'DRAFT',
+      fixtures.enduserGuardianFixture,
+      fixtures.enduserChildFixtureKaarina,
+      fixtures.daycareFixture.id
+    )
+  ])
+
+  page = await (await newBrowserContext({ acceptDownloads: true })).newPage()
+
+  await insertEmployeeFixture({
+    id: config.financeAdminAad,
+    externalId: `espoo-ad:${config.financeAdminAad}`,
+    email: 'lasse.laskuttaja@espoo.fi',
```

Comment:
Supernit: I guess this doesn't need to a real address -> I'd prefer something definitely non-existent like `billy.biller@billing.test` (`.test` will [never resolve](https://www.rfc-editor.org/rfc/rfc6761.html#section-6.2)), just to avoid the unnecessary risk of extra spam to espoo.fi